src/models/token.js

```diff
@@ -0,0 +1,28 @@
+const { Sequelize, DataTypes } = require('sequelize');
+
+const { User } = require('./user');
+const { sequelize } = require('../utils/db.js');
+
+const Token = sequelize.define(
+  'Token',
+  {
+    refreshToken: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    expireAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: Sequelize.literal(`NOW() + INTERVAL 30 DAYS`),
```

Comment:
The use of `Sequelize.literal` with `NOW() + INTERVAL 30 DAYS` is not valid. Consider using Sequelize's built-in date functions or setting the default value in a different manner, such as using a hook to set the expiration date.